propolis/src/hw/rtc.rs

```diff
@@ -53,8 +53,7 @@ impl Rtc {
         let high_bytes = high_chunks.to_le_bytes();
         hdl.rtc_write(MEM_OFF_HIGH, high_bytes[0])?;
         hdl.rtc_write(MEM_OFF_HIGH + 1, high_bytes[1])?;
-        // XXX Shouldn't this be "high_bytes[2]"?
-        hdl.rtc_write(MEM_OFF_HIGH + 2, high_bytes[1])?;
+        hdl.rtc_write(MEM_OFF_HIGH + 2, high_bytes[2])?;
```

Comment:
Can probably just commit this by itself (which would mean little more context included about the fix too, rather than being lumped in with all your docs work)

Reply:
Done